cli/daemon/dash/dashapp/src/components/app/AppAPI.tsx

```diff
@@ -21,12 +21,13 @@ export default class AppAPI extends React.Component<Props, State> {
     this.onNotification = this.onNotification.bind(this)
   }
 
-  async componentDidMount() {
+  componentDidMount() {
```

Comment:
Having `async componentDidMount` made the `componentWillUnmount` function run before the `componentDidMount`. Not great if we want to remove the event listener.

Also, We should really move away from the class based React components for a bunch of reasons.